### PR TITLE
Removed Multiple providers section, replaced with note

### PR DESCRIPTION
**Pull request by molebox**

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/02-data-sources.mdx [hunk 1]
```diff
@@ -20,6 +20,8 @@ A Prisma schema can only have _one_ data source. However, you can:
 - [Programmatically override a data source `url` when creating your `PrismaClient`](../../../reference/api-reference/prisma-client-reference#programmatically-override-a-datasource-url)
 - [Specify a different URL for Prisma Migrate's shadow database if you are working with cloud-hosted development databases](../prisma-migrate#cloud-hosted-shadow-databases-must-be-created-manually)
 
+> **Note**: Multiple provider support was removed in 2.22.0. Please see [Deprecation of provider array notation](https://github.com/prisma/prisma/issues/3834) for more information.
```
Comment (Jolg42) [inline content/200-concepts/100-components/01-prisma-schema/02-data-sources.mdx:23]:
```suggestion
> **Note**: Multiple providers support was removed in 2.22.0. Please see [Deprecation of provider array notation](https://github.com/prisma/prisma/issues/3834) for more information.
```

**Comment by molebox** [comment]
Im just merging this

**Comment by janpio** [comment]
2.22 will only be released tomorrow

**Comment by molebox** [comment]
> 2.22 will only be released tomorrow

Oops! Hmm, that will teach me for being impatient 😬

**Comment by janpio** [comment]
Wheeee: https://github.com/prisma/prisma/releases/tag/2.22.0

**Pull request merged**